src/Oscoin/Consensus.hs

```diff
@@ -34,8 +34,8 @@ validateBlockchain validateBlock (Blockchain (blk :| blks)) =
     validateBlock blks blk *> validateBlockchain validateBlock (fromList blks)
 
 
--- | Validates the size (in bytes) for a block, comparing the serialised size
--- with the maximum allowed value as read from the configuration file.
+-- | Validates the size (in bytes) of a block, comparing the serialized size
```

Comment:
so are we standardising on US spelling? :trollface:

Reply:
Yeah aka "International English" :thinking: